### PR TITLE
Bombs

### LINKED ISSUE #9: Bomb Class
A Class needs to be created for a bomb.

Basic Class requirements
- Position: Most likely a Vector2I
- Countdown: A float to keep track of time left over
- Update function: Removes delta time from countdown returning a bool indicating and explosion.

Functionality could be expanded on this issue. Once a bomb explodes there will need to be some way of indicating on the map that there is now fire in the cell as well as logic checks to destroy walls.

Current collision logic uses the map to check neighbouring cells. As such we will most likely need to update the when a bomb is placed to insure player cannot enter a cell with a bomb

### PR DESCRIPTION
**Pull request by charlchi**
Closes #9        

Fixes errors from #20 : The variables in player.moveState weren't being initialised to false, causing the player to move.

### CODE REVIEW

**Comment by charlchi** [comment]
I also changed the player to move more slowly so that slower machines don't experience clipping as heavily while we're still using prototype graphics

File: renderer/src/Renderer.cpp [hunk 1]
```diff
@@ -1,17 +1,41 @@
...
+	circle.setFillColor(sf::Color(250, 20, 50));
+	
+    std::list<Bomb> bombs = state.bombs.bomb_list;
+    std::list<Bomb>::iterator  i = bombs.begin();
+    while (i != bombs.end())
+    {
+		sf::Vector2i position((*i).position);
+		circle.setPosition((*i).position.x * SCALE,
+			(*i).position.y * SCALE);
+		if ((fmod((*i).tick, 0.4)) > 0.2)
```
Comment (WSeegers) [inline renderer/src/Renderer.cpp:32]: This is game logic in the renderer

File: CMakeLists.txt [hunk 2]
```diff
@@ -1,7 +1,7 @@
 cmake_minimum_required(VERSION 3.0)
 
 project(nibbler)
-set(CMAKE_CXX_STANDARD 17)
+set(CMAKE_CXX_STANDARD 11)
```
Comment (WSeegers) [inline CMakeLists.txt:4]: You need to sort out your machine to use 17. All updated versions of gcc and clang support it.

**Review by WSeegers — CHANGES_REQUESTED** [review]
Create a separate ticket for these changes. Stating addition features you have implemented or bugs that you have fixed.

**Comment by WSeegers** [comment]
Im going to close this pull request due to the fact that it's currently piggy backing on the previous pull request implementing bombs. Open up a ticket referencing issues. This might sound tedious but as an example it seem like you have implemented destruction of blocks but it isn't noted anywhere. Also conditional redering is logic that does not belong in a render class.

**Pull request closed without merging**